packages/react-native-platform-override/src/e2etest/Api.test.ts

```diff
@@ -36,10 +36,6 @@ test('hasOverride', async () => {
   await usingRepository('sampleOverrideRepo', async (_, repoPath) => {
     const manifestPath = path.join(repoPath, 'overrides.json');
 
-    expect(
```

Comment:
We should leave these test cases/files.
This isn't a real override of the yoga.cpp file, but a test of the react-native-platform-override package's override system, testing that overriding a file works correctly.